tests/testdata/default_config.yaml

```diff
@@ -19,7 +19,8 @@ chat:
   # Filepath of a dialog session file.
   # Default: None
   session:
-  # Controls the randomness of the model's responses.
+  # Controls the randomness of the model's responses. Lower values make the output
+  # more deterministic, while higher values produce more random results.
```

Comment:
this extra line isn't related to the PR though?

Reply:
`make regenerate-testdata` regenerates the entire default cfg, so someone must have modified the temp help text but not run this cmd so my PR is picking that up